### ISSUE TITLE
Can not reproduce result for visual place recognition

### ISSUE THREAD

**Issue by EvelinYin** [Red-Fairy/ZeroShotDayNightDA#6]
Hi there. I am trying to train the model for visual place recognition. As in the code, the pretrained daytime model is just borrowed from the pretrained resnet101 for ``` darkening.py ```. Then for ```train_night.py ```, since I didn't find the resume checkpoint specified [here](https://github.com/Red-Fairy/ZeroShotDayNightDA/blob/45a7862521d23abcfe49545653062337b10ff8b2/visual-place-recognition/cirtorch/examples/train_night.py#L140), I just disabled loading from the checkpoint, and the code starts training using the pretained resnet101 as default. All other configurations are the default ones specified as in the Readme file (Besides that I have added ```--sim``` flag which I think you missed in the instruction).

However, I failed to reproduce the result for this task. In fact, during the 5 training epochs, the reported MAP is decreasing (0.799 down to 0.67) with the decrease of the loss, which is really confusing. Could you help to figure out which part I have missed or is doing wrong? Thanks a lot.

**Comment by Red-Fairy** [comment]
You may find the post-adaptation model in the updated [link](https://drive.google.com/drive/folders/1E1BizpMh-G-eJVISHJ0mw8717ryAnul3?usp=sharing). Besides, be aware that the train-time evaluation result is not identical to that of test-time. Please follow the original [repo](https://github.com/Attila94/cnnimageretrieval-pytorch/tree/ea6ac81e26e46fa2f58e6ca376607bf6f12350a4) for the correct evaluation practices.

**Comment by EvelinYin** [comment]
> You may find the post-adaptation model in the updated [link](https://drive.google.com/drive/folders/1E1BizpMh-G-eJVISHJ0mw8717ryAnul3?usp=sharing). Besides, be aware that the train-time evaluation result is not identical to that of test-time. Please follow the original [repo](https://github.com/Attila94/cnnimageretrieval-pytorch/tree/ea6ac81e26e46fa2f58e6ca376607bf6f12350a4) for the correct evaluation practices.

Hi thanks so much for your timely reply! I do have followed the evaluation process and have got the reported MAP(90.4%) on the post-adaptation model ```visual_place_recog_post_adaptation.tar``` you provided [here](https://drive.google.com/drive/folders/1E1BizpMh-G-eJVISHJ0mw8717ryAnul3?usp=sharing).
 
However, when I try to train from scratch by myself (pretrained resnet101 as init for both [darkening](https://github.com/Red-Fairy/ZeroShotDayNightDA/blob/7aafb399931c6587e705b92d96d7d2effd47b7a5/darkening/darken_vpr.py#L99) and [train_night](https://github.com/Red-Fairy/ZeroShotDayNightDA/blob/7aafb399931c6587e705b92d96d7d2effd47b7a5/visual-place-recognition/cirtorch/examples/train_night.py#L202) ), but I only got MAP to be 83.66%.
![image](https://github.com/Red-Fairy/ZeroShotDayNightDA/assets/42159046/25cfa401-d4c3-427f-887f-4cec2256d32c)



Is it because the missing checkpoint specified in the script [here](https://github.com/Red-Fairy/ZeroShotDayNightDA/blob/45a7862521d23abcfe49545653062337b10ff8b2/visual-place-recognition/cirtorch/examples/train_night.py#L140)? Should I just simply increase the number of epochs for training in the ```train_night.py```?

**Comment by Red-Fairy** [comment]
Hi,

Thanks for pointing out the mistake. As mentioned in our paper, our model is adapted from a task-specific model pre-trained on the corresponding daytime dataset, i.e., we first train the model (initialized by resnet-101) using the daytime data only (script `train.py`), then adapt it to nighttime. I have updated the `resume` argument to `pretrained_model_path` to avoid misunderstanding.
